wsclient/src/main/java/org/hpccsystems/ws/client/utils/Connection.java

```diff
@@ -158,7 +158,9 @@ public Connection(String connectionstring) throws MalformedURLException
         setPort(Integer.toString(theurl.getPort()));
         setURIPath(theurl.getPath());
 
-        options = theurl.getQuery().split("&");
+        options = new String[0];
```

Comment:
perhaps options should be initialized to null,

Reply:
@rpastrana initialized to null